### ISSUE TITLE
CRNN Line based From epoch1 to epoch 140 the loss remains almost the same and accuracy is 0.0000.

### ISSUE THREAD

**Issue by soufieneghribi** [PaddlePaddle/PaddleOCR#5039]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
- I am training a line-level model for arabic sentences recognition. I am at epoch 140. The loss started from high to 60~70. The loss remains between 40~70 for several epochs and the accuracy is always 0.0000.
- I am using the arabic_dict and 500.000 synthetic line segments. I tired the same config with words it's working well but with sentences it gives random results.
- Can someone please help me?

- 系统环境/System Environment：Linux
- 版本号/Version：PaddlePaddle 2.2.0, compiled with
    with_avx: ON
    with_gpu: ON
    with_mkl: ON
    with_mkldnn: ON
    with_python: ON
  PaddleOCR：latest
 问题相关组件/Related components：
- 运行指令/Command Code：python3 -m paddle.distributed.launch --gpus '1' tools/train.py -c  rec_arabic_light.yml

**Comment by GanWang** [comment]
同样的问题，求教

**Comment by tink2123** [comment]
When you train with a single card, you should use a smaller learning rate. You can try to adjust the learning rate to 0.0005 or use [pretrained_model](https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/arabic_ppocr_mobile_v2.0_rec_infer.tar)

**Comment by MohieEldinMuhammad** [comment]
@soufieneghribi 
Can you share your experience on training on sentence level, did it wokr well ? What main changes i need to make ? 
Also can you please share the tool you used to generate the data.
